src/uct/ib/base/ib_md.c

```diff
@@ -885,7 +881,7 @@ static ucs_status_t uct_ib_mkey_pack(uct_md_h uct_md, uct_mem_h uct_memh,
     if ((memh->flags & UCT_IB_MEM_ACCESS_REMOTE_ATOMIC) &&
         !(memh->flags & UCT_IB_MEM_FLAG_ATOMIC_MR) &&
         (memh != &md->global_odp) &&
-        md->config.enable_umr_pack)
+        md->config.enable_indirect_atomic)
```

Comment:
is there another place where enable_indirect_atomic is used? maybe this check here should be removed at all?

Reply:
it should be enabled by default... how can we remove this check?

Reply:
maybe https://github.com/openucx/ucx/blob/master/src/uct/ib/base/ib_md.c#L196 is sufficient?

Reply:
then how we cover case when user force disable indirect atomics (even it is supported)?

Reply:
looks to me the according to that code in umr_qp_create, if md->config.enable_indirect_atomic==0 it behaves as if it's unsupported

Reply:
done